src/components/date_picker/date_picker.tsx

```diff
@@ -89,6 +102,11 @@ interface EuiExtendedDatePickerProps extends ReactDatePickerProps {
    * Show the icon in input
    */
   showIcon?: boolean;
+
+  /**
+   * Open the popper in one of the positions
+   */
+  popperPlacement?: EuiDatePickerPopperPlacement;
```

Comment:
Can we name this more similar to our own naming conventions for this type of `pop` component:
```suggestion
  popoverPlacement?: EuiDatePickerPopperPlacement;
```